test/controllers/test_classes_controller.rb

```diff
@@ -208,7 +208,7 @@ def test_roots_for_cls
     roots = MultiJson.load(last_response.body)
     assert_equal 9, roots.length
     roots.each do |r|
-      assert_instance_of String, r["prefLabel"]
+      assert_instance_of String, r["prefLabel"], r["@id"]
```

Comment:
why this changes?

Reply:
when this assertion fails (for example if prefLabel = nil), it's informative to know the ID of the term that caused this failure